### DISCUSSION TITLE
Is there a way to change default feed settings?

### DISCUSSION THREAD

**Discussion by glottisfaun0000** [FreshRSS/FreshRSS#5833] [Q&A] [answered]
I'd like to make the filter action "Mark an article as read if an identical title already exists in the top n newest articles" set to a value by default for newly added feeds - is this possible?

**Comment by Alkarex** [comment]
There is a global option, and then you can disable it at feed level (or the other way around)

**Reply by glottisfaun0000** [reply, marked as answer]
Ah- found it in `Conguration > Reading > Mark article as read`. Thanks!
